src/components/bookSupport.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+// import { useSelector } from 'react-redux';
+
```

Comment:
- [ ] Kindly 😊 try to remove the unnecessary comment here because this makes your codebase ugly ,so we want your code to look nice now and in the next projects try to not put unnecessary comments at the base code to make the code organized and nice for the others😃